fdbserver/ClusterController.actor.cpp

```diff
@@ -3071,6 +3071,8 @@ class ClusterControllerData {
 	Counter registerMasterRequests;
 	Counter statusRequests;
 
+	Reference<EventCacheHolder> clusterControllerEventHolder;
```

Comment:
It's possible in the future we will have more of these in the cluster controller. I would suggest giving this a name associated with the event (e.g. `recruitedMasterWorkerEventHolder`) to avoid confusion.